cbr-rule-module/main.tf

```diff
@@ -34,6 +34,7 @@ resource "ibm_cbr_rule" "cbr_rule" {
           operator = attribute.value.operator
         }
       }
+      # Access tags for the target resources.  These resource tags specified in the rule should match to the tags attached to the target service access tags.
```

Comment:
It's unclear to me, are the tags you set when creating the service access tags or service tags? I expect they are service tags and if so do CBRs work with them or only Access tags?  If it only works with access tags we should be more explicit. https://cloud.ibm.com/apidocs/tagging#filtering-tags

Reply:
CBR works only with Access tags.  In UI access tags are part of the creation of service  & in terraform we need to create using "ibm_resource_tag" with type "access".  

I understand the confusion is because of the word "resource tags" in the comment.  Will update it.